### DISCUSSION TITLE
First Discussion

### DISCUSSION THREAD

**Discussion by marcelinorc** [robs74/offices-generate-money#5] [General]
Hello Robs74!

This is TheFunKat from Steam.

I see there is a lot going on on Steam about this script!

I propose we coordinate tasks based on issues on this repo. 

How about you (the leader of the project) create issues with the ideas of the forum and then we modify the script based on those issues?

Thanks!
TheFunKat

**Comment by robs74** [comment]
Yeah, that's basically what I've done.
